CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Features ✨ and improvements 🏁
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
+* Add experimental line-edge-clip property. ([1679](https://github.com/mapbox/mapbox-maps-android/pull/1679))
```

Comment:
Perhaps remove as we will be wrapping that?

Reply:
yeah, agree, will remove for now and re-introduce when we create platform wrappings.